### ISSUE TITLE
ChildProcess incorrect types

### ISSUE THREAD

**Issue by rdavison** [sikanhe/reason-node#47]
It appears `Process` and `ChildProcess` modules' `stdin`, `stdout`, and `stderr` values are all the opposite type from what they should be.

**Comment by banacorn** [comment]
Yes, `stdout` and `stderr` should be *writable* while `stdin` should be *readable*

**Comment by banacorn** [comment]
wondering if this repo is still being maintained

**Comment by TheSpyder** [comment]
> Yes, `stdout` and `stderr` should be _writable_ while `stdin` should be _readable_

isn't that already how it works?
https://github.com/sikanhe/reason-nodejs/blob/b7b41349f85665d6cd7fc76a170229428e7ec94c/src/ChildProcess.re#L81-L86

I'm working on a fork that swaps to ReScript syntax and fixes any niggling little issues, but this one seems to have always been correct.
[edit] now released as [rescript-nodejs](https://github.com/TheSpyder/rescript-nodejs)

**Comment by TheSpyder** [comment]
I see how I was wrong. Fixed in thespyder/rescript-nodejs#4.
